Eastern-Research-Group/mywaterway
# Feature/379 cyan panel edits 2

## Related Issues:
* [HMW-379](https://jira.epa.gov/secure/RapidBoard.jspa?rapidView=864&view=detail&selectedIssue=HMW-379)

## Main Changes:
* Updated the style and organization of the CyAN panel.
* Updated text descriptions in the CyAN panel and the Monitoring tab.
* Added responsiveness to the column chart so that it scales in the map popup.
* Added "Very Low" and "Unknown" series to the stacked column chart.

## Steps To Test:
1. Navigate to [Lake Mattamuskeet](http://localhost:3000/community/030201050101/monitoring), then expand the _Lake Mattamuskeet_ accordion item.
2. Confirm that the edits specified in _cyan-edits-3.pdf_, attached to the Jira ticket, are implemented.
3. Confirm that the _Minimum Value_ and _Average and Standard Deviation_ fields have been removed.
4. Confirm that tooltips have been added beside the _Waterbody Area_ and _Satellite Image Pixel Area_ rows.
5. Click the CyAN waterbody on the map, and verify that the charts and their corresponding text are scaled to fit in the popup.
